projects/magento/pageObjects/checkout/ShippingDetails.page.js

```diff
@@ -6,17 +6,17 @@ export class ShippingDetails {
 
     // Shipping Section Locators
     this.emailInput = page.locator(
-      "//*[@id='customer-email' and @type='email' and @aria-required='true']"
+      "#checkout-step-shipping #customer-email"
     );
-    this.firstNameInput = page.locator("input[name='firstname']");
-    this.lastNameInput = page.locator("input[name='lastname']");
-    this.addressInput = page.locator("input[name='street[0]']");
-    this.countrySelector = page.locator("select[name='country_id']");
-    this.stateProvinceDropdown = page.locator("select[name='region_id']");
-    this.stateProvinceField = page.locator("input[name='region']");
-    this.cityInput = page.locator("input[name='city']");
-    this.postCodeInput = page.locator("input[name='postcode']");
-    this.phoneNumberInput = page.locator("input[name='telephone']");
+    this.firstNameInput = page.locator("#checkout-step-shipping input[name='firstname']");
```

Comment:
Should we move `#checkout-step-shipping` to a constant?

Reply:
Makes sense, done!